internal/system/executor/metrics.go

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package executor
+
+import (
+	"fmt"
+	"strings"
+)
+
+const separator = ";"
+
+func metricsExecutorCommands(serviceName string) []string {
+	return []string{
+		"stats",
+		serviceName,
+		"--no-stream",
+		"--format",
+		"{{ .CPUPerc }}" + separator +
```

Comment:
:+1: thanks this is exactly the kind of thing I was hoping to see

Reply:
I'm pleased that you're pleased. ;)